### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by amprokop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,15 +268,6 @@ When a VA employee logs in through the VA's unified login system (CSS) a session
 Within this session the user gets a set of roles. These roles determine what pages a user has access to.
 In dev mode, we don't log in with CSS and instead take on the [identity of a user in the database](#changing-between-test-users).
 
-## Dispatch (Dev Mode)
```
Comment (amprokop) [inline README.md]: @sharonwarner cool if i remove this bit? it seems outdated

**Comment by amprokop** [comment]
good thought, @kahlouie— trying to keep this PR small for now but i'm definitely open to adding that later!

File: README.md [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+### Queue
+
+Workflow management at the Board of Veterans' Appeals.
+
+### Reader
+
+Increases the speed with which attorneys and Veterans Law Judges (VLJs)
+review and annotate electronic case files.
+
+### Hearings Schedule
```
Comment (cmgiven) [inline README.md]: Hearing Schedule

File: README.md [hunk 3]
```diff
@@ -1,13 +1,50 @@
...
+
+Enables the Board to schedule Veterans for hearings.
```
Comment (cmgiven) [inline README.md]: Scheduling and supporting Board of Veterans' Appeals hearings.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,50 @@
...
+
+## Caseflow products in a mature state 
+
+### Dispatch 
+
+Facilitates the transfer of cases from the Agency of Original Jurisdiction (AOJ) to
+the Board of Veterans' Appeals (the Board).
+
+### Hearings
```
Comment (cmgiven) [inline README.md]: Hearing Prep

File: README.md [hunk 5]
```diff
@@ -1,13 +1,50 @@
...
+
+Improving the timeliness and Veteran experience of Board hearings.
+
+### API
+
+Providing Veterans transparent information about the status of their appeal
+
+### Certification
+
+Facilitates the transfer of cases from the Agency of Original Jurisdiction (AOJ) to
```
Comment (cmgiven) [inline README.md]: Extra line break.

File: README.md [hunk 6]
```diff
@@ -1,13 +1,50 @@
 # Caseflow
 [![CircleCI](https://circleci.com/gh/department-of-veterans-affairs/caseflow.svg?style=svg)](https://circleci.com/gh/department-of-veterans-affairs/caseflow)
 
-Clerical errors have the potential to delay the resolution of a veteran's appeal by **months**. Caseflow Certification uses automated error checking, and user-centered design to greatly reduce the number of clerical errors made when certifying appeals from offices around the nation to the Board of Veteran's Appeals in Washington DC.
+Caseflow is a suite of web-based tools to manage VA appeals. It's currently in development by the Appeals Modernization team (est. 2016). It will replace the current system of record for appeals, the Veterans Appeals Control and Location System (VACOLS), which was created in the late 1980s on now-outdated infrastructure. Additionally, Caseflow will allow the Board of Veterans' Appeals to process appeals under the new guidelines created by the Appeals Modernization Act of 2017, which goes into effect February 14th, 2019.
```
Comment (cmgiven) [inline README.md]: The product impact statement refers to the "current incarnation" of VACOLS, which was created in the late 1980s. If referring to VACOLS itself, it began life in 1979.

If writing out AMA, it's the Veterans Appeals Improvement and Modernization Act of 2017.

File: README.md [hunk 7]
```diff
@@ -1,13 +1,50 @@
...
 
-[You can read more about the project here](https://medium.com/the-u-s-digital-service/new-tool-launches-to-improve-the-benefits-claim-appeals-process-at-the-va-59c2557a4a1c#.t1qhhz7h8).
+The Appeals Modernization team's mission is to empower employees with technology to increase timely, accurate appeals decisions and improve the Veteran experience. Most of the team's products live here, in the main Caseflow repository.
```
Comment (cmgiven) [inline README.md]: Should we also link to the eFolder Express repo? To Feedback? Anything else?

**Pull request merged**